argo-e2e/Dockerfile.buster

```
@@ -0,0 +1,23 @@
+FROM python:slim-buster
+
+RUN set -ex \
+    && apt update \
```

I would also add upgrade for security patches
```suggestion
    && apt update \
    && apt upgrade -y \
```